src/App.js

```diff
@@ -22,7 +22,6 @@ const initialState = {
   mapPosition: [],
   mapPositionHistory: [],
   isModalOpen: true,
-  loading: true,
```

Comment:
Can you explain the reason for this?

Reply:
Leaving a comment to resolve this as I explained it further in the pr